### ISSUE TITLE
where is the code

### ISSUE THREAD

**Issue by jiyuay** [speech2face/speech2face.github.io#3]
Hello,this study is cool! But in the repository I can only find the wav file and image of a person. where will the code be available?
Thank you so much!

**Comment by adam0white** [comment]
I believe they have concerns about usage of it before releasing the code

**Comment by 12944qwerty** [comment]
I agree with @abdulvahidisik . This project has some privacy and security concerns. It may be sometime

**Comment by salif** [comment]
I am still waiting for the code

**Comment by dashimaki360** [comment]
I am still waiting for the code
